examples/js/gatsby/gatsby-browser.js

```diff
@@ -0,0 +1,46 @@
+import Bugsnag from "@bugsnag/js"
+import React from "react"
+import BugsnagPluginReact from '@bugsnag/plugin-react'
+
+const bugsnagConfig = {
+    // env vars prefixed with `GATSBY_` are made available to the browser
+    apiKey: process.env.GATSBY_BUGSNAG_API_KEY,
+    plugins: [new BugsnagPluginReact()],
+    appVersion: '1.2.3',
+}
+
+/**
+ * With `gatsby develop` wrapRootElement is called before onClientEntry and then again afterwards.
+ * However, `gatsby serve` onClientEntry is called, followed by wrapRootElement. So we need this 
+ * extra state to deal with the difference in behavior.
+ */
+let bugsnagStarted = false
+
+export const onClientEntry = () => {
+    if (bugsnagStarted) {
+        return
+    }
+    Bugsnag.start(bugsnagConfig)
+    bugsnagStarted = true
+}
```

Comment:
This can use [`Bugsnag.isStarted`](https://github.com/bugsnag/bugsnag-js/blob/7dac9382ced03e31cabb8ba9a0ae8546f75d6e08/packages/browser/src/notifier.js#L86-L88) instead of keeping track manually

```suggestion
export const onClientEntry = () => {
    if (Bugsnag.isStarted) {
        return
    }

    Bugsnag.start(bugsnagConfig)
}
```